Lib/test/test_os.py

```diff
@@ -179,6 +179,13 @@ def test_tmpnam(self):
             else:
                 self.check_tempfile(name)
 
+    @unittest.skipUnless(sys.platform == 'win32', 'tests windows behaviour')
+    def test_listdir_extended_length_path(self):
+        """See https://bugs.python.org/issue32539"""
+        dirname = '\\\\?\\' + os.path.abspath(test_support.TESTFN)
+        result = os.listdir(dirname)
+        self.assertEqual(result, [])
```

Comment:
Does it matter if the directory empty? What if other files are left after other tests? Is it worth to test non-empty directory?

Reply:
I believe the `setUp` / `tearDown` ensure this is empty, the contents of the directory don't actually matter, just that it exists and that the `listdir(...)` call does not crash.  I added the assertion so it looks less like a useless test :)